config/core/test-configs.yaml

```diff
@@ -767,6 +767,12 @@ test_plans:
     params:
       job_timeout: '20'
 
+  wifi-basic: &wifi-basic
```

Comment:
To other reviewers: although this anchor is not used anywhere yet, it's a placeholder for more complex wifi test, see e.g. `v4l2-decoder-conformance` in L698